integr8ly/integreatly-operator
# Consolidate RHSSO

# Description
Consolidate RHSSO

# How to Verify
The verification is essentially a full regression on RHSSO Cluster and User

- Create a test-user in group rhmi-developer (./scripts/setup-htpass-idp.sh)
- Confirm you can log into OpenShift and then 3Scale and Solution Explorer
- Log into RHSSO User as the test-user
- Log into RHSSO Cluster as admin
